plugins/tekton/src/routes.ts

```diff
@@ -1,5 +1,11 @@
-import { createRouteRef } from '@backstage/core-plugin-api';
+import { createRouteRef, createSubRouteRef } from '@backstage/core-plugin-api';
 
 export const rootRouteRef = createRouteRef({
   id: 'tekton',
 });
+
+export const pipelineRunRouteRef = createSubRouteRef({
+  id: 'pipeline-run-visualization',
+  parent: rootRouteRef,
+  path: '/cluster/:clusterName/pipelineRun/:pipelineRunName',
```

Comment:
here it says `:clusterName` but what's passed in number i.e index https://github.com/janus-idp/backstage-plugins/pull/506/files#diff-cd65d6b98bf0c3daff8d2edaaad8e013f7a55c35d6df69374fca9a4c571c710eR28

Reply:
`clusters[selectedCluster as number]` is the name of the selected cluster

Reply:
ah my bad, thanks 👍